app/assets/javascripts/discourse/app/models/topic-list.js

```diff
@@ -92,10 +92,11 @@ const TopicList = RestModel.extend({
 
       return ajax({ url: moreUrl }).then((result) => {
         let topicsAdded = 0;
+        let newTopics = [];
```

Comment:
I'm not sure if we need this change?

Reply:
It needs to be in this scope to allow the `return` line below to see it (unlike Ruby, `if` does have its own scope in JS)

Reply:
The `return` below is within the `if` scope as well so we shouldn't have to declare the variable outside of the scope.

Reply:
Looks like I need glasses lol, will fix it thanks!